src/components/Navbar.tsx

```diff
@@ -22,7 +23,7 @@ export default function Navbar() {
 
       <div className={navRevealedState?'Menu':'Menu menu-hidden'}>
         <a href="/">Home</a>
```

Comment:
```suggestion
        <Link href="/">Home</Link>
```
I think this needs to be a `<Link/>` element too?